src/components/ServiceCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardFooter
+} from "@baltimorecounty/dotgov-components";
+
+let starImg = {
+  fontSize: ".2em",
+  float: "right",
+  marginTop: "0px",
+  width: "10px"
+};
+
+const ServiceCard = ({ name, department, url, icon, rank }) => (
+  <div className="d-flex col-lg-4 col-md-6 col-sm-6">
+    <Card>
+      <CardContent>
+        <div className="dg_icon-container">
+          <i className={icon} aria-hidden="true"></i>
```

Comment:
I am not seeing this icon, but i can see the most popular icon.

Reply:
@martypowell sorry this was bad data in the structured content. This has been fixed.

Reply:
thanks tim, yep, clearing the service cache resolved this.